packaging/nnstreamer.spec

```diff
@@ -36,6 +36,10 @@
 %define armnn_support 0
 %endif
 
+%ifarch %arm aarch64
+%define vivante_support 0
+%endif
```

Comment:
This is weird. Are you leaving it undefined intentionally for other archs?

Reply:
Currently, Vivante supports only ARM32 and ARM64. 
(In case of other archs, when we verify the operation and compatibility, other achs can be added.)